app/src/main/res/values/strings.xml

```diff
@@ -32,4 +32,31 @@
     <string name="room_members">Members</string>
     <string name="room_sync">sync</string>
     <string name="kick_member">Kick member</string>
+
+    // Downloads UI Strings
+
+    <string-array name="downloads_name">
+        <item>Download 1</item>
+        <item>Download 2</item>
+        <item>Download 3</item>
+    </string-array>
+
+    <string-array name="downloads_status">
+        <item>Downloading</item>
+        <item>Completed</item>
+        <item>Downloading</item>
+    </string-array>
+
+    <string-array name="downloads_speed">
+        <item>1Kbps</item>
+        <item>2Kbps</item>
+        <item>3Kbps</item>
+    </string-array>
+
+    <string-array name="downloads_percentage">
+        <item>10%</item>
+        <item>20%</item>
+        <item>30%</item>
+    </string-array>
```

Comment:
Strings is generally used for localization etc. For testing I'd suggest to use a dummy class which returns needed data.